VK-GL-CTS's check_build_sanity.py build error

On VK-GL-CTS master the script check_build_sanity.py returns an error when building Amber on GCC 64 bits. Amber's HEAD there is: 34c750892a8a8bf7e3440ea25cf4c8c690a44827

My GNU/Linux system has GCC v8.3.0 (Debian Buster). The error is the following:

```
-- Build files have been written to: /tmp/deqp-build-test/gcc-64-release
[76/1464] Building CXX object external/amber/src/src/CMakeFiles/libamber.dir/vkscript/command_parser.cc.o
FAILED: external/amber/src/src/CMakeFiles/libamber.dir/vkscript/command_parser.cc.o 
/usr/bin/g++  -DAMBER_CTS_VULKAN_HEADER=1 -DAMBER_ENABLE_SHADERC=0 -DAMBER_ENABLE_SPIRV_TOOLS=0 -DAMBER_ENGINE_DAWN=0 -DAMBER_ENGINE_VULKAN=1 -DDE_ASSERT_FAILURE_CALLBACK -DDE_COMPILER=DE_COMPILER_GCC -DDE_CPU=DE_CPU_X86_64 -DDE_OS=DE_OS_UNIX -DDE_PTR_SIZE=8 -I/home/siglesias/devel/vk-gl-cts/external/amber/../vulkancts/framework/vulkan -I/home/siglesias/devel/vk-gl-cts/framework/delibs/debase -I/home/siglesias/devel/vk-gl-cts/framework/delibs/decpp -I/home/siglesias/devel/vk-gl-cts/framework/delibs/depool -I/home/siglesias/devel/vk-gl-cts/framework/delibs/dethread -I/home/siglesias/devel/vk-gl-cts/framework/delibs/deutil -I/home/siglesias/devel/vk-gl-cts/framework/delibs/destream -I/home/siglesias/devel/vk-gl-cts/external/vkrunner/src -I/home/siglesias/devel/vk-gl-cts/external/spirv-headers/src/include -I/home/siglesias/devel/vk-gl-cts/external/amber/src/include -I/home/siglesias/devel/vk-gl-cts/external/amber/src -I. -Wall -Wextra -Wno-long-long -Wshadow -Wundef -Wconversion -Wno-sign-conversion -Werror -Wno-error=unused-function -Wno-implicit-fallthrough -m64 -std=c++03 -Wno-delete-non-virtual-dtor -fwrapv -fvisibility=hidden -fvisibility-inlines-hidden -std=c++11 -fvisibility=hidden -Wall -Werror -Wextra -Wno-unknown-pragmas -Wpedantic -pedantic-errors -O3 -DNDEBUG -fPIC   -fno-exceptions -fno-rtti -std=gnu++11 -MD -MT external/amber/src/src/CMakeFiles/libamber.dir/vkscript/command_parser.cc.o -MF external/amber/src/src/CMakeFiles/libamber.dir/vkscript/command_parser.cc.o.d -o external/amber/src/src/CMakeFiles/libamber.dir/vkscript/command_parser.cc.o -c /home/siglesias/devel/vk-gl-cts/external/amber/src/src/vkscript/command_parser.cc
In file included from /home/siglesias/devel/vk-gl-cts/external/amber/src/src/vkscript/command_parser.h:24,
                 from /home/siglesias/devel/vk-gl-cts/external/amber/src/src/vkscript/command_parser.cc:15:
/home/siglesias/devel/vk-gl-cts/external/amber/src/src/command.h: In member function ‘amber::Result amber::vkscript::CommandParser::ProcessProbeSSBO()’:
/home/siglesias/devel/vk-gl-cts/external/amber/src/src/command.h:309:53: error: ‘comp’ may be used uninitialized in this function [-Werror=maybe-uninitialized]
   void SetComparator(Comparator comp) { comparator_ = comp; }
                                         ~~~~~~~~~~~~^~~~~~
/home/siglesias/devel/vk-gl-cts/external/amber/src/src/vkscript/command_parser.cc:2052:32: note: ‘comp’ was declared here
   ProbeSSBOCommand::Comparator comp;
                                ^~~~
cc1plus: all warnings being treated as errors
```

--- a/src/vkscript/command_parser.cc
+++ b/src/vkscript/command_parser.cc
@@ -2049,7 +2049,7 @@ Result CommandParser::ProcessProbeSSBO() {
     return Result("Invalid comparator for probe ssbo command: " +
                   token->ToOriginalString());
 
-  ProbeSSBOCommand::Comparator comp;
+  ProbeSSBOCommand::Comparator comp = ProbeSSBOCommand::Comparator::kEqual;
   r = ParseComparator(token->AsString(), &comp);
   if (!r.IsSuccess())
     return r;

